R/support_panel.R

```diff
@@ -136,35 +135,32 @@ support_panel <- function(
         "The parent statistical release of this dashboard, along with
               methodological information,
               is available at the following link: ",
```

Comment:
I would drop some of this and change to:

`is available at [explore education statistics](link).`